gapir/cc/android/asset_resource_cache.cpp

```diff
@@ -48,13 +48,16 @@ bool asset_read(AAsset *asset, void *buf, size_t count) {
   return true;
 }
 
-// touch pages will touch all memory pages for a given memory span
+// touch_pages will touch all memory pages for a given memory span
```

Comment:
We should either update this comment, or slightly change the logic here. 

"touch_pages will write at least one 0 onto every page in the given memory span"

Alternatively we should read and write a the same value, so that at least the contents of the pages are unchanged.